### PR TITLE
test: improve CSV import route coverage

### PR DESCRIPTION
**Pull request by RudraMalvankar**
## Purpose

Improve edge-case test coverage for the CSV import route.

## Tasks

* [x] add test for empty request body handling
* [x] add test for unsupported content-type handling
* [x] verify Vitest tests pass locally

## Verify

* Run:

  ```bash
  pnpm exec vitest run src/import/csv.test.ts
  ```

* Run:

  ```bash
  pnpm exec vitest run --coverage
  ```

## Before

N/A (test-only change)

## After

N/A (test-only change)

### CODE REVIEW

File: src/import/csv.test.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+            mockDataSource,
+            mockConfig
+        )
+
+        expect(response.status).toBe(400)
+    })
+})
+
+it('returns 400 for unsupported content type', async () => {
+    const request = new Request('http://localhost', {
```
[the remark this answers is not in the document]
Comment (RudraMalvankar) [reply src/import/csv.test.ts]: Addressed in the latest commit — moved tests into the same describe block and removed the unintended workspace config file.

File: src/import/csv.test.ts [hunk 2]
```diff
@@ -0,0 +1,48 @@
+import { describe, it, expect, vi } from 'vitest'
+import { importTableFromCsvRoute } from './csv'
+
+const mockDataSource = {} as any
+const mockConfig = {} as any
+
+vi.mock('../export', () => ({
+    executeOperation: vi.fn(),
+}))
```
[the remark this answers is not in the document]
Comment (RudraMalvankar) [reply src/import/csv.test.ts:5]: Addressed in the latest commit — moved tests into the same describe block and removed the unintended workspace config file.
